src/drivers/optical_flow/pmw3901/PMW3901.cpp

```diff
@@ -324,8 +324,8 @@ PMW3901::RunImpl()
 		return;
 	}
 
-	delta_x = (float)_flow_sum_x / 500.0f;		// proportional factor + convert from pixels to radians
-	delta_y = (float)_flow_sum_y / 500.0f;		// proportional factor + convert from pixels to radians
+	delta_x = (float)_flow_sum_x / 385.0f;		// proportional factor + convert from pixels to radians
```

Comment:
Likely applies to the paw3902 as well.

Reply:
@dagar I thought the same but don't have one to verify that; Are you sure that the scaling is the same?